src/staking-v3/hooks/useDapps.ts

```diff
@@ -35,6 +35,10 @@ export function useDapps() {
     try {
       aggregator.publish(new BusyMessage(true));
       const dApps = await service.getDapps(currentNetworkName.value.toLowerCase());
+      const numberOfParticipants = await service.getNumberOfParticipants(
```

Comment:
I have a concern here. If this call fails dApp staking UI will be unusable because dapps are not stored to Vuex. Please move participants fetching to the very end of the method.